sms/send-with-slim/index.php

```diff
@@ -10,8 +11,8 @@
         return $response->withStatus(400)->write("No message provided");
     }
 
-    $client = new Nexmo\Client(new Nexmo\Client\Credentials\Basic(API_KEY, API_SECRET));
-    $text = new \Nexmo\Message\Text($args['number'], NEXMO_FROM_NUMBER, $body['text']);
+    $client = new Nexmo\Client(new Nexmo\Client\Credentials\Basic(NEXMO_API_KEY, NEXMO_API_SECRET));
+    $text = new \Nexmo\Message\Text($args['number'], FROM, $body['text']);
```

Comment:
When `config.php` is included it iterates over `$_ENV` and defines a constant for each key. This is how `NEXMO_API_KEY` and `NEXMO_API_SECRET` have values.

Following that, `FROM` should be `FROM_NUMBER` as that's the name in `.env`

Reply:
I think it's `.env` that wants updating then, `FROM` is our standard placeholder.